src/views/Planning/components/AssignmentRow.tsx

```diff
@@ -0,0 +1,144 @@
+
+
+export const AssignmentRow = ({ index, setSelectedAssignment }: {
+  index: number
+  setSelectedAssignment: React.Dispatch<React.SetStateAction<number | undefined>>
+}): JSX.Element => {
+  const { get: getTitle } = useYObserver('meta', `core/assignment[${index}]`)
+  const { get: getUUID } = useYObserver('meta', `core/assignment[${index}].links.core/article[0]`)
+  const { get: getAssignmentDescription } = useYObserver('meta', `core/assignment[${index}].meta.core/description[0].data`)
+  const { get: getAssignmentPublishTime } = useYObserver('meta', `core/assignment[${index}].data`)
+  const { state: stateAuthor = [] } = useYObserver('meta', `core/assignment[${index}].links.core/author`)
+  const [showVerifyDialog, setShowVerifyDialog] = useState<boolean>(false)
+
+  return (
+    <div className="grid grid-cols-12 border-b py-2">
+
+      <div className="col-span-8 px-2 py-1">
+        <div className="flex-grow flex space-x-2 items-center">
+          <div className='font-medium text-sm'>
+            {getUUID('uuid')
+              ? <Link to='Editor' props={{ id: getUUID('uuid') as string }}>
+                {(getTitle('title') as Y.XmlText)?.toJSON()}
+              </Link>
+              : <span className='text-muted-foreground'>{(getTitle('title') as Y.XmlText)?.toJSON()}</span>}
+          </div>
+
+          <SluglineEditable path={`core/assignment[${index}].meta.tt/slugline[0]`} />
+        </div>
+
+        <div className='font-normal text-sm mt-2'>{(getAssignmentDescription('text') as Y.XmlText)?.toJSON()}</div>
+      </div>
+
+      <div className="col-span-4 flex justify-end space-x-4 items-center">
+        <AssigneeAvatars assignees={Array.isArray(stateAuthor) ? stateAuthor.map((author) => author.name) : []} />
+
+        <AssignmentType index={index} />
+
+        <div className="min-w-[64px] whitespace-nowrap">
+          {getAssignmentPublishTime('publish')
+            ? <TimeDisplay date={new Date(getAssignmentPublishTime('publish') as string)} />
+            : '-'
+          }
+        </div>
+
+        <div className="whitespace-nowrap">
+          <DotDropdownMenu
+            items={[
+              {
+                label: 'Redigera',
+                icon: Edit,
+                item: () => setSelectedAssignment(index)
+              },
+              {
+                label: 'Ta bort',
+                icon: Delete,
+                item: () => {
+                  setShowVerifyDialog(true)
+                }
+              }
+            ]}
+          />
+        </div>
+      </div>
+
+      {showVerifyDialog &&
+        <VerifyDeleteAssignmentDialog
+          index={index}
+          title={(getTitle('title') as Y.XmlText)?.toJSON()}
+          setSelectedAssignment={setSelectedAssignment}
+          setShowVerifyDialog={setShowVerifyDialog}
+        />
+      }
+    </div>
+  )
+}
+
+
+function VerifyDeleteAssignmentDialog({ index, title, setSelectedAssignment, setShowVerifyDialog }: {
```

Comment:
Picky, but shouldn't it be `const VerifyDeleteAssignmentDialog () =>` for style reasons